### PR TITLE
Enable Chrome OS screenshot [stage-7]

### PR DESCRIPTION
**Pull request by andrecardoso**

### CODE REVIEW

**Comment by andrecardoso** [comment]
[(5) ChrOS: display screenshot](https://trello.com/c/DOXa176g/4215-5-chros-display-screenshot)

File: test/unit/displays/services/svc-player-pro-factory.tests.js [hunk 1]
```diff
@@ -139,6 +139,10 @@ describe('service: playerProFactory:', function() {
     expect(playerProFactory.isScreenshotCompatiblePlayer({playerName: 'RisePlayerElectron', playerVersion:''})).to.be.false;
     expect(playerProFactory.isScreenshotCompatiblePlayer({playerName: 'RisePlayerElectron', playerVersion:'2016.06.27.05.15'})).to.be.false;
     expect(playerProFactory.isScreenshotCompatiblePlayer({playerName: 'RisePlayerElectron', playerVersion:'2017.07.31.15.31'})).to.be.true;
+    expect(playerProFactory.isScreenshotCompatiblePlayer({playerName: 'RisePlayerElectron', playerVersion:'2017.07.31.15.31'})).to.be.true;
```
Comment (alex-deaconu) [inline test/unit/displays/services/svc-player-pro-factory.tests.js]: @andrecardoso this test is a duplicate from the one above and should probably be removed. Thanks.

**Comment by alex-deaconu** [comment]
@andrecardoso looks good 👍 , just one comment inline. Thanks.

File: test/unit/displays/services/svc-player-pro-factory.tests.js (hunk 1 above)
Comment (andrecardoso) [reply test/unit/displays/services/svc-player-pro-factory.tests.js]: @alex-deaconu good catch! Removed.

**Comment by alex-deaconu** [comment]
@andrecardoso looks good, thanks for fixing 👍

**Pull request merged**